mappings/net/minecraft/NetworkPathNode.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS net/minecraft/class_4459 net/minecraft/NetworkPathNode
```

Comment:
Renamed, though I don't properly remember why I mapped this. I think it was because it had some interactions with the debug classes but if you're already mapped them in a PR I can probably leave it out of this one.

Reply:
Which PR did you map them in?